framework/src/actions/SetAdaptivityOptionsAction.C

```diff
@@ -162,7 +162,7 @@ SetAdaptivityOptionsAction::act()
       if (!isParamSetByUser("disable_p_refinement_for_families"))
         // If the user has not set this parameter we will set a logicial default
         disable_p_refinement_for_families =
-            "LAGRANGE NEDELEC_ONE LAGRANGE_VEC CLOUGH BERNSTEIN RATIONAL_BERNSTEIN";
+            "LAGRANGE NEDELEC_ONE RAVIART_THOMAS LAGRANGE_VEC CLOUGH BERNSTEIN RATIONAL_BERNSTEIN";
```

Comment:
I thought I would need to tell you to add this here but you found it on your own. Much appreciated!

Reply:
No worries. :)